### PR TITLE
Redesign of the blog card #17

### PR DESCRIPTION
**Pull request by devyaniChoubey**

### CODE REVIEW

**Comment by thebishaldeb** [comment]
As I can see from the commits, you solved multiple issues in this PR. Actually I assigned the edit and add comments to someone else. So can't give you points for that. And also solve an issue only when I approve it and it is labelled in any of the categories.

And please provide screenshots of the new pagews that were designed.

File: models/usermodel.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ var mongoose    = require("mongoose"),
 
 var UserSchema  = new mongoose.Schema({
     username    : String,
-    password    : String
```
Comment (thebishaldeb) [inline models/usermodel.js:6]: Add age, gender, description (which would be a small paragraph about him), birthdate, role (which would be admin/user).

**Comment by thebishaldeb** [comment]
Also comment on the issues before solving them and make sure one PR has only one issue solved. Multiple issues should not be solved in a single PR because if something goes wrong and we need t revert back to the precious state, then all the commits will be gone from that PR.

**Comment by devyaniChoubey** [comment]
sir there were several changes in the project after you merged last pr of vk gupta .
Images were flowing out of the div.So i fixed it.

**Comment by devyaniChoubey** [comment]
> Also comment on the issues before solving them and make sure one PR has only one issue solved. Multiple issues should not be solved in a single PR because if something goes wrong and we need t revert back to the precious state, then all the commits will be gone from that PR.



> As I can see from the commits, you solved multiple issues in this PR. Actually I assigned the edit and add comments to someone else. So can't give you points for that. And also solve an issue only when I approve it and it is labelled in any of the categories.
> 
> And please provide screenshots of the new pagews that were designed.

Screenshot (7)
Screenshot (8)

**Comment by thebishaldeb** [comment]
> sir there were several changes in the project after you merged last pr of vk gupta .
> Images were flowing out of the div.So i fixed it.

Yeah that's fine. Create issues, I'll assign them to you and thus you will get points. Resolving them volunteeringly would not get you points since there would not be any track of that.

**Comment by devyaniChoubey** [comment]
Sir will u merge this pr?

**Comment by thebishaldeb** [comment]
I'll give you points for the #17  #6  after you increase the schema.

Also the edit and add comment redesigning I meant the form and someone else is working on that. 

Give a screenshot of edit blog page and I'll give you points for that as well.

**Comment by thebishaldeb** [comment]
> Sir will u merge this pr?

Yeah, just make those changes and push the commit, I'll merge it.

**Comment by devyaniChoubey** [comment]
Screenshot (11)

**Pull request merged**